statsmodels/discrete/discrete_model.py

```diff
@@ -2525,25 +2528,31 @@ def _score_nbin(self, params, Q=0):
         y = self.endog[:,None]
         mu = self.predict(params)[:,None]
         a1 = 1/alpha * mu**Q
+
+        prob = a1 / (a1+mu)
```

Comment:
you could still use prob but define it as

prob = 1 / (1 + alpha * mu**(1-Q))
where I used 1 / a1 instead a1  which is well defined and numerically stable at and around alpha=0

Reply:
I'd be fine with this if there were a comment to the effect of "# Note: Q is either 1 or 0".  A reviewer had to ask about that recently.

Reply:
AFAICS, this holds for every Q.
However, code comment about Q would be helpful, because the name is uninformative, and I always got confused about Q versus P in NegBin.
e.g. Q=0 for nb2 and Q=1 for nb1 or maybe it's reversed :(

Reply:
I think it was non-obvious because of the substitution `mu/alpha` --> `a1` in a location conditioned on `if Q:` as opposed to `if Q == 1:`.